app/services/idv/flows/in_person_flow.rb

```diff
@@ -55,11 +55,14 @@ def self.session_idv(session)
       end
 
       def extra_analytics_properties
-        return {} if @flow_session[:pii_from_user][:same_address_as_id].nil?
-        {
-          same_address_as_id: @flow_session[:pii_from_user][:same_address_as_id].to_s == 'true',
+        extra = {
           pii_like_keypaths: [[:same_address_as_id], [:state_id, :state_id_jurisdiction]],
         }
+        if @flow_session[:pii_from_user].has_key?(:same_address_as_id)
```

Comment:
Looks like this made one of the tests fail. I suggest we add back the `nil` check here as such:

```suggestion
        unless @flow_session[:pii_from_user]&.[](:same_address_as_id).nil?
```

Reply:
May also be good to update the test to cover this.